### DISCUSSION TITLE
[help] I can't get Telegram notifications to work

### DISCUSSION THREAD

**Discussion by GreatestFool** [vogler/free-games-claimer#230] [Q&A]
I've read [this thread](https://github.com/vogler/free-games-claimer/discussions/59#discussion-4852930) and I've checked the documentation in the README as well as the one over [appraise](https://github.com/caronc/apprise/wiki/Notify_telegram) but for the life of me, I just cannot figure it out. When it runs, I just get this kind of error: `error: Command failed: apprise tgram://<bottoken(chatID:tokenID>/<chatID> -i html '-t TEST' -b 'epic-games: got captcha during login. Please check.'`

In the `docker-compose.yml` I've set it up under `environment:` as `NOTIFY: tgram://<bottoken(chatID:tokenID>/<chatID>`. The parenthesis is just there to explain/show the structure, since everyone's so vague on it that I find it hard to follow lol. So mostly for my own sake. Does anyone know what I'm doing wrong?

<details>
<summary>docker-compose.yml</summary>

```yaml
services:
  free-games-claimer:
    container_name: vogler-fgc # is printed in front of every output line
    image: ghcr.io/vogler/free-games-claimer # otherwise image name will be free-games-claimer-free-games-claimer
    build: .
    restart: unless-stopped
    network_mode: bridge
    ports:
      - "6080:6080" # noVNC (browser-based VNC client)
    volumes:
      - /volume1/docker/fgc:/fgc/data
    command: bash -c "node epic-games;"
    environment:
      #SHOW: 0
      WIDTH: 1280
      HEIGHT: 720
      NOTIFY: <bottoken(chatID:tokenID>/<chatID>
      NOTIFY_TITLE: "TEST"
      EG_EMAIL: [REDACTED]
      EG_PASSWORD: [REDACTED]
      EG_OTPKEY: [REDACTED]
```

</details>

**Comment by duckfacejones** [comment]
I have mine as

environment:
      - NOTIFY='tgram://XXXX:XXXX/XXX'
